images/commons/Dockerfile

```diff
@@ -26,8 +26,6 @@ RUN apk update \
     && mkdir -p /lagoon/crontabs && fix-permissions /lagoon/crontabs \
     && ln -s /home/.bashrc /home/.profile
 
-RUN chmod g+w /etc/passwd
```

Comment:
do we not do this here because we assume no-one runs commons, only inherits, or is there another reason?

Reply:
yes exactly. I don't think we run commons anywhere, do we?

Reply:
is there any harm in keeping this here (or using it's own fix-permissions?) - on the off-chance a derivative image doesn't fix permissions?

Reply:
no, you're right there's no harm in leaving this here. I'll update the PR :+1: